auto-instrument-handler/lumigo_wrapper

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
rename to `auto-instrument-handler/lumigo_wrapper.sh` ?

Reply:
I prefer it to be without the `.sh` because that's how they wrote their examples (all our special extensions scripts looks like that) 🤔